src/Router.php

```diff
@@ -504,27 +509,26 @@ public static function process_http_request() {
 
 		// Previously there was a small distinction between the response and the result, but
 		// now that we are delegating to Request, just send the response for both.
-		$result = $response;
```

Comment:
Instead of hiding this inline in a function, i'm noting the extraneous usage in each place it's relevant.